src/models.py

```diff
@@ -171,9 +174,45 @@ def build_model(args, vocab, pretrained_embs, tasks):
     for task in tasks_to_build:
         # If the name of the task is different than the classifier it should use
         # then skip the module creation.
-        if task.name != model._get_task_params(task.name).get('use_classifier', task.name):
+        if task.name != model._get_task_params(
+                task.name).get(
+                'use_classifier',
+                task.name):
+            log.info(
+                "Name of the task is different than the classifier it should use")
             continue
-        build_module(task, model, d_sent, d_emb, vocab, embedder, args)
+        build_task_specific_components(
+            task, model, d_sent, d_emb, vocab, embedder, args)
+
+def build_model(args, vocab, pretrained_embs, tasks):
+    '''Build model according to args
+    Returns: model which has attributes set in it with the attrbutes.
+
+    '''
+
+    # Build embeddings.
+    if args.openai_transformer:
+        # Note: incompatible with other embedders, but logic in preprocess.py
+        # should prevent these from being enabled anyway.
+        from .openai_transformer_lm.utils import OpenAIEmbedderModule
+        log.info("Using OpenAI transformer model; skipping other embedders.")
+        cove_layer = None
+        # Here, this uses openAIEmbedder.
+        embedder = OpenAIEmbedderModule(args)
+        d_emb = embedder.get_output_dim()
+    else:
+        # Default case, used for ELMo, CoVe, word embeddings, etc.
+        d_emb, embedder, cove_layer = build_embeddings(args, vocab,
+                                                       tasks, pretrained_embs)
+    d_sent = args.d_hid
+
+    sent_encoder, d_sent = build_sent_encoder(
+        args, vocab, d_emb, tasks, embedder, cove_layer)
+
+    d_sent += args.skip_embs * d_emb
```

Comment:
Comment as to what this does?

Reply:
This increases the input dimension of the task-specific module to become encoder_input_dimension + encoder_output_dimension (if args.skip_embs = 1).